tests/python/test_genotypes_api.py

```diff
@@ -0,0 +1,206 @@
+import mock
+import nose
+import nose.tools as asserts
+import json
+
+from cycledash import db
```

Comment:
Do we have a story around line breaks between imports? Personally, I've been generally doing 1 group of imports for internal and (with an extra line break in between the two groups) 1 group of imports for external